code/modules/supply/supply_pack.dm

```diff
@@ -32,3 +32,29 @@
 		//Add the name to the UI manifest
 		ui_manifest += "[initial(AM.name)]"
 	manifest += "</ul>"
+
+/datum/supply_packs/proc/create_package(spawn_location)
```

Comment:
Maybe scope spawn location to an atom

Reply:
Good idea, but maybe a turf would be better. I dont think we should ever be spawning crates inside lockers